HW_1/Task_3_Numbers.py

```diff
@@ -0,0 +1,21 @@
+# Задача 3. Напишите код, который запрашивает число и сообщает является ли оно простым или составным. 
+# Используйте правило для проверки: “Число является простым, если делится нацело только на единицу и на себя”. 
+# Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+
+a_num = -1
+
+while a_num < 0 or a_num > 100_000:
+    a_num = int(input("Введите положительное число не больше 100 000: "))

```

Comment:
Можно проще
```suggestion
while True:
    a_num = int(input("Введите положительное число не больше 100 000: "))
    if 0 < a_num < 100_000
        break
```
или (но хуже)
```suggestion
while not 0 < (a_num = int(input("Введите положительное число не больше 100 000: ")) < 100_000:
    pass
```